lcpp.lua

```diff
@@ -1377,9 +1375,36 @@ end
 -- @param predefines OPTIONAL a table of predefined variables
 -- @usage out, state = lcpp.compileFile("../odbg/plugin.h", {["MAX_PAH"]=260, ["UNICODE"]=true})
 function lcpp.compileFile(filename, predefines, macro_sources, next, _local)
-	if not filename then error("processFile() arg1 has to be a string") end
-	local file = io.open(filename, 'r')
-	if not file then error("file not found: "..filename) end
+	if type(filename) ~= 'string' then error("processFile() arg1 has to be a string") end

```

Comment:
Absolutely, C_INCLUDE_PATH and CPLUS_INCLUDE_PATH are not standard include environment variables or something. Maybe something some IDE brings in. But this is not the right place to 'fix' this.